internal/compiler/wasm/wasm.go

```diff
@@ -3,6 +3,7 @@
 // license that can be found in the LICENSE file.
 
 // Package wasm contains an IR->WASM compiler backend.
+// nolint: deadcode,varcheck // Package in development (2021).
```

Comment:
😄 I'll take care of that when this is merged.

Reply:
Great! I tried to make sure that if the linters found "dead code" that had been touched in the past year, that I turned off the linter so it wouldn't bug us.